pretty_benchmark_result.ts

```diff
@@ -21,7 +21,7 @@ export type prettyBenchmarkResultOptions =
 const c: Colorer = new Colorer();
 
 export function prettyBenchmarkResult(
-  options: prettyBenchmarkResultOptions,
+  options?: prettyBenchmarkResultOptions,
```

Comment:
you could do 

`options: prettyBenchmarkResultOptions = {}`


if you want to set defaults, in the body of the method - 
```ts
const defaultOptions = {
  nocolor: false,
};

// Then inside prettyBenchmarkResult
const opts = {...defaultOptions, ...options};
```

Then any user option would overload default.

Reply:
I like the `?` better, but I know, that the current solution eg: https://github.com/littletof/prettyBenching/blob/bdaedc125ad5f002d7ad2ae3af72ba48c037e157/benchmark_result_card.ts#L49-L61 is very ugly. I may do a refactor, where I use the spreading instead.

Reply:
yeah, I am supportive =)

Reply:
Noted :)